nvflare/private/fed/app/simulator/simulator_worker.py

```diff
@@ -119,6 +119,7 @@ def release_resources(self, client):
         self.logger.info(f"Clean up ClientRunner for : {client.client_name} ")
 
     def run(self, args, conn):
+        self.logger.info("ClientTaskWorker started to run")
```

Comment:
If 1000 clients, will we see 1000 of these messages?
Is this meaningful for regular users?

Reply:
it's useful info for each client run.